jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramWidgetView.html

```diff
@@ -14,5 +14,5 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<div class="container-fluid" id="processDiagramDiv" style="overflow:scroll;height:600px;width:800px;">
+<div class="container-fluid" id="processDiagramDiv" style="overflow:auto; height:600px">
```

Comment:
@nmirasch minor comment here but I believe you can also remove the fixed height.